### DISCUSSION TITLE
Jetty working but Sage Web Interface is dead

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#110] [General]
I went to upgrade some of the web related plugins and somehow I have killed by SageTV web UI.  Jetty still works, and some of the Jetty web pages work - Sage Mobile works, SageX works, , apidocs, and bmt all work.  My extenders can connect and run so miniclients can run fine.  It is just the /sage/Home and livepvr pages that aren't working.

When I go to the web page Jetty gives me a 404 error.   I tried uninstalling and reinstalling the SageTV Web Interface plugin.  Anyone have any other ideas on why this would have happened?  I am running Sage under unRAID and this server has run fine for years.
 
`<html><body>
<h2>HTTP ERROR 404 Not Found</h2>

URI: | /sage/Home
-- | --
404
Not Found
-

<hr><a href="https://eclipse.org/jetty">Powered by Jetty:// 9.4.44.v20210927</a>
</body>
</html>HTTP ERROR 404 Not Found
URI:	/sage/Home
STATUS:	404
MESSAGE:	Not Found
SERVLET:	-
[Powered by Jetty:// 9.4.44.v20210927](https://eclipse.org/jetty)`

**Comment by jusjoken** [comment]
What do you get if you go to /apps.

**Reply by wayner9** [reply]
I also get the same Jetty 404 error at /apps.  

But for the mobile apps I am ok at http://192.168.1.99:7070/sagem/m/ and bmt at http://192.168.1.99:7070/bmt/#status

I also tried this on new browsers/PCs that haven't hit this site before to ensure that it wasn't a caching issue.

**Comment by jusjoken** [comment]
Perhaps a missing dependency on the update you did.  Can you post a SageTV
log to review as I imagine it should indicate a failing plugin  dependency.

k

**Reply by wayner9** [reply]
The Sagetv_0.txt log, or one of the Jetty or webserver logs?

**Comment by jusjoken** [comment]
My guess is its a failing plugin which will show in the SageTV log

**Reply by wayner9** [reply]
Here are my logs, all zipped up
[sagetv_logs.zip](https://github.com/OpenSageTV/Community/files/13851795/sagetv_logs.zip)
